requirements.txt

```diff
@@ -1,4 +1,3 @@
-ipython==3.1.0.dev0
```

Comment:
Is there a reason for removing this (rather than I guess changing the version number)?